gemfiles/actionpack_5.0.gemfile.lock

```diff
@@ -58,6 +63,23 @@ GEM
       rake (>= 0.8.7)
       thor (>= 0.18.1, < 2.0)
     rake (10.5.0)
+    rspec-core (3.8.0)
```

Comment:
These changes resulted from my last PR to add rspec support. As Rails 5 tests weren't running at the time, I didn't update the lock until now.